### DISCUSSION TITLE
Olimex ESP32-POE version G

### DISCUSSION THREAD

**Discussion by thermseekr** [arendst/Tasmota#14144] [Q&A] [answered]
Hello all,

Has anyone tried the new version G of the Olimex ESP32-POE board? I got 4 of them, uploaded Tasmota V10.0.0 but i can get no connection neither through wifi nor through ethernet. Wifi actually works as I can access the AP mode interface to select an SSID, but after connecting to the network and getting an IP from the DHCP server it goes mute. Cannot ping nor access the web interface.

Ethernet also no good, as I don't even get a link on the switch port.

I know version G uses a PoE chip from Texas as the usual Si3204 from Silicon Labs is not available, but I'm not aware of any other mods that would impact communications.

Any thoughts?

**Comment by s-hadinger** [comment]
I have the old model. I'm surprised that wifi fails. Any logs when connecting serial? And what template did you try?

**Reply by AmyYummy** [reply]
Did you buy it from an [online store](www.baudcom.cn)?

**Comment by sfromis** [comment]
Looks like Si3204 is not among the supported, unless it is compatible with one of:
```
0 = ETH_PHY_LAN8720, 
1 = ETH_PHY_TLK110/ETH_PHY_IP101, 
2 = ETH_PHY_RTL8201, 
3 = ETH_PHY_DP83848, 
4 = ETH_PHY_DM9051, 
5 = ETH_PHY_KSZ8081
```
Of course, this should not prevent you from getting on wifi. What do you see on serial console while it is trying to connect?

**Reply by Jason2866** [reply]
The LAN ETH_PHY is still a LAN8720

**Comment by thermseekr** [comment]
Serial logs are normal. It connects to the SSID, I can see it getting an IP address from DHCP, and I see status messages as usual. But, no access at all to the web interface, no ping, no nothing. So I didn’t get to the point to configure a template.

**Comment by thermseekr** [comment]
Si3204 is the old chip, the one they could no longer get, so they’re now using another chip from Texas, I don’t know which one. But, AFAIK this is the PoE chip, not the Ethernet one. So even if it’s bad I should be able to power the board through USB or the 5V pin. I tried everything and I cannot get Wi-Fi to communicate and Ethernet won’t even link.

Enviado do meu iPhone

**Comment by s-hadinger** [comment, marked as answer]
From olimex website:
The new revisions of the boards are REV.G and the PCB info will be updated on GitHub tomorrow. From user point of view nothing changes – the new revision operates exactly the same and have same functionality as the previous Silicon Labs based solution.

**Reply by thermseekr** [reply]
Yes I have read that. So maybe I just have problematic units. Actually, from the 4 boards I bought, one cannot communicate over USB, one keeps resetting, and the remaining two are the ones I could test and have the problem I described.

I'll do further testing tomorrow, I hope I can identify the problem as I really need two working boards.

I have an older version of the ISO model that works like a charm. That's why I thought it's version related.

Cheers
Tales

**Comment by thermseekr** [comment]
Ok, finally got them working. I had more than one problem, so it's hard to pinpoint exactly what happened. Finally after a lot of tweaking in my config overrides I managed to compile and upload a working firmware.

About the ethernet link, I totally forgot that I would never get one before configuring the corresponding GPIOs, or applying a template.

And, of the four ESP32-POE boards that I got, only two are actually working. The third is in a reset loop (watchdog) and the last one has a non working USB port.

It was a long couple of days.

Merry Xmas everyone!
Tales
